libraries/AP_HAL_ChibiOS/hwdef/BROTHERHOBBYH743/hwdef.dat

```diff
@@ -158,6 +158,7 @@ PD2 SDMMC1_CMD SDMMC1
 # VTX Power control - should be high at startup to ensure power
 PD10 VTX_PWR OUTPUT HIGH GPIO(81)
 define RELAY2_PIN_DEFAULT 81
+define RELAY2_FUNCTION 1
```

Comment:
My reading of the code is that this doesn't work, either needs changes to AP_Relay or needs to go in defaults.parm